scripts/3_visualize/visualizeLandUse.R

```diff
@@ -65,11 +67,12 @@ visualizeLandUse <- function(fname.data, fname.fig){
                              meanFiber = "orange",
                              meanFilm = "yellow",
                              meanFoam = "blue"))
-  
+
   gs.conc <- gsplot() %>% 
     rect(position.df.conc$x.left, position.df.conc$y.bottom, 
          position.df.conc$x.right, position.df.conc$y.top,
-         lwd=0.5, col = position.df.conc$rect.col) %>% 
+         lwd=0.5, col = position.df.conc$rect.col, 
+         legend.name=levels(position.df.conc$type)) %>% 
     axis(side = 1, at = position.df.conc$x.middle, 
          labels = position.df.conc$site.name, 
          tick = FALSE, las = 2, cex.axis = 0.1) %>% 
```

Comment:
does this function run for you? seems you save `saveRDS(gs_conc, file = "cache/visualize_conc.RDS")` but gs_conc is called gs.conc